src/Common/src/Interop/Windows/user32/Interop.EnumWindows.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class User32
+    {
+        internal delegate bool EnumThreadWindowsCallback(IntPtr hWnd, IntPtr lParam);
+
+        [DllImport(Libraries.User32)]
```

Comment:
@gkhanna79 @weshaggard what's our policy for Pinvoke declarations. By "moving away from API sets" I assume we mean "assuming reverse forwarders are available"

I note that src\interop\windows\interop.libraries.cs includes both API sets and regular DLL's so I don't know that we're consistent in the code at all.